### DISCUSSION TITLE
vim打开文本会随机有部分字符乱码-bug问题

### DISCUSSION THREAD

**Discussion by htmljsp** [Eugeny/tabby#7007] [Q&A]
<img width="722" alt="image" src="https://user-images.githubusercontent.com/7731411/185387267-0d54ed2c-ab5b-48b6-95cb-6c42d32725ae.png">

**Comment by lawweiliang** [comment]
@htmljsp 
English only please.
https://github.com/Eugeny/tabby/discussions/5533

![image](https://user-images.githubusercontent.com/34936929/228172727-cc58dcc4-bdf4-4085-844a-a25cf784c145.png)
